wallet/src/main/java/com/android/identity_credential/wallet/PresentationActivity.kt

```diff
@@ -119,91 +103,36 @@ class PresentationActivity : FragmentActivity() {
         application as WalletApplication
     }
 
-    private var deviceRequest: ByteArray? = null
+    // device request bytes
+    private var deviceRequestByteArray: ByteArray? = null
     private var deviceRetrievalHelper: DeviceRetrievalHelper? = null
-    private var consentData: ConsentPromptEntryFieldData? = null
-
-    // Transfer helper facilitates starting processing a presentation request and obtaining the
-    // response bytes once processing has finished. This enables showing one or more dialogs to
-    // the user to accept before sending a response to requesting party.
-    private var transferHelper: TransferHelper? = null
-
-    // Define the Builder for building a TransferHelper once it gets new instance of DeviceRetrievalHelper
-    private val transferHelperBuilder: TransferHelper.Builder by lazy {
-        TransferHelper.Builder(
-            settingsModel = walletApp.settingsModel,
-            documentStore = walletApp.documentStore,
-            trustManager = walletApp.trustManager,
-            context = applicationContext,
-            onError = showErrorAndDismiss
-        )
-    }
 
-    // lambda called whenever an error is encountered during processing of the Presentation request
-    private val showErrorAndDismiss: (Throwable) -> Unit = { throwable ->
-        Toast.makeText(applicationContext, throwable.message, Toast.LENGTH_SHORT).show()
-        onDestroy()
-    }
+    // TODO: add SD_JWT_VC support
+    // the supported formats for the Credentials of a Document
+    val supportedCredentialFormats = listOf(CredentialFormat.MDOC_MSO)
```

Comment:
I only brought it here because when I cleaned up `TransferHelper` for removal, the existing TODO's (from you) asked for these changes - to process multiple docRequests and support more credential formats.

<img width="974" alt="Screenshot 2024-06-21 at 10 40 49 PM" src="https://github.com/openwallet-foundation-labs/identity-credential/assets/1477025/07d74298-93c8-480d-bfb2-21986f61ca7c">

Initially I defined `supportedCredentialFormats` inside the function's signature with a default value (and not a **val**)

```
  fun isDocumentSuitableForDocRequest(
            document: Document,
            docType: String,
            supportedCredentialFormats: List<CredentialFormat> = listOf(CredentialFormat.MDOC_MSO)
        ): Boolean { 
        ...
        }
```

This provides a default value for the 3rd parameter and it does not need to be provided when calling the function (unless you want to check for additional or non-mdoc-mso credential formats). 

I think I'll nest `supportedCredentialFormats ` back in the function's signature and give the option to the caller to provide their own list of supported credential formats (esp. useful for testing) if they wanted to but by default it is mdoc mso.